apps/dashboard/widgets.py

```diff
@@ -107,10 +107,16 @@ def render(self, name, value, attrs=None):
         else:
             final_attrs['value'] = ''
 
+        if attrs.get('placeholder', False):
```

Comment:
This is not it. And even if it was, it shouldn't have mattered since if a field has a value, that will have higher priority than the placeholder of the field. The placeholder is hidden if the field value is set.